dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -0,0 +1,190 @@
+# Use a single iot-lab node with RIOT
+# ===================================
+#
+# Supported:
+#  * flash
+#  * reset
+#  * term
+#
+# Tested on m3/a8-m3/wsn430/samr21/arduino-zero nodes
+#
+# It can be run:
+# * From your computer by setting IOTLAB_NODE to the full url like
+#   m3-380.grenoble.iot-lab.info or a8-1.grenoble.iot-lab.info
+# * From IoT-LAB frontend by setting IOTLAB_NODE to the short url like
+#   m3-380 or a8-1
+#
+# # Usage
+#
+#   make BOARD=iotlab-m3 IOTLAB_NODE=m3-380.grenoble.iot-lab.info flash term
+#
+# It is the user responsability to start an experiment beforehand
+#
+# If the user has multiple running experiments, the one to use must be
+# configured with the `IOTLAB_EXP_ID` setting
+#
+# Prerequisites
+# -------------
+#
+# * Install iotlab-cli-tools: https://github.com/iot-lab/cli-tools
+# * Configure your account with 'iotlab-auth'
+# * Register your computer public key in your ssh keys list on your profile:
+#   https://www.iot-lab.info/tutorials/ssh-cli-client/
```

Comment:
I think this link should go with the use of "A8 node from a frontend section" below. The correct tutorial for setting the SSH access is here: https://www.iot-lab.info/tutorials/configure-your-ssh-access/

Reply:
Ahhhh yeah.

Reply:
I knew I would miss something when I did it :D